services-js/commissions-app/src/client/common/Checkbox.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export interface CheckboxProps {
+  name: string;
+  id: any;
+  value: any;
+  onChange: any;
+  onBlur: any;
+  title: string;
+}
+
+export default class Checkbox extends React.Component<CheckboxProps> {
+  render() {
+    return (
+      <div>
+        <label className="cb" htmlFor="Checkbox-${this.props.name}">
+          {this.props.title}
```

Comment:
Take a look back at https://patterns.boston.gov/components/detail/checkbox.html to see the HTML you need to use to make the text appear in the right place.